usagov/usagov-2021
# USAGOV-839-Spanish-Meta-Field: Update field.field.node.basic_page.fie…

…ld_meta_description.yml


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-839

## Description

**Import config changes** Config change to bring "meta description" to every basic page even if it has a language toggle.

### Testing Instructions
Before this change:
-  localhost -> edit: you see the "meta description"
- localhost/es -> edit: you **do not** see the "meta description"
You can see this field in the page source:  `<meta name="description" `

After this change:
-  localhost -> edit: you see the "meta description"
- localhost/es -> edit: you **do** see the "meta description"
You can see this field in the page source:  `<meta name="description" `

## Checklist for the Developer



## Checklist for the Peer Reviewers
